content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,226 @@
+active: false
+capacity: 20
+priority: 1
+range: [151-170]
+reset: false
+ldap: false
+location: 'mougins'
+replayId: 31
+varpass: false
+compile: false
+workshopImg: 'https://us-central1-grommet-designer.cloudfunctions.net/images/frederic-passeron-hpe-com/WOD-GO-101-A-simp-introduction-to-Go-programming-language.jpeg'
+badgeImg: 'https://us-central1-grommet-designer.cloudfunctions.net/images/frederic-passeron-hpe-com/go101-a-simple-introduction-to-go-programming-language.jpg'
+beta: false
+category: 'Open Source'
+duration: 4
+alternateLocation: 'grenoble'
+presenter: 'Matthew Doddler'
+role: 'FullStack developer'
+avatar: '/img/SpeakerImages/MattD.jpg'
+videoLink: 'https://hpe-developer-portal.s3.amazonaws.com/Workshops-on-Demand-Coming-Soon-Replay.mp4'
+```
+
+A﻿s the developer of the Workshops-on-demand content, Matt had to perform several tasks:
+
+##### O﻿n the backend server:
+
+1. ###### Log on to the backend server and clone the notebook repo in his home folder or as explained earlier fork the repo on his laptop and work from there.
+2. ###### Create a new branch for his workshop following the naming convention defined with the admin.
+3. ###### L﻿everage the template provided by me to build up the content of his workshop.
+4. ###### T﻿est the workshop locally on his laptop or on the dev server leveraging the `test-action.sh` script
+5. ###### T﻿est the workshop using the staging registration portal.
+6. ###### W﻿hen all tests are green, create a pull request to merge content with master repo.
+
+A﻿s an admin, I need to check the pull request and accept it. Once done, the test/dev and staging environments will require an update.
+
+1. ###### Log on to the backend server as wodadmin and update the notebook repository.
+2. ###### R﻿un a wod-deliver to update the relevant backend server.
+
+   ```
+   git remote update
+   git rebase
+   wod-deliver
+   ```
+
+T﻿he very same processes will apply to the move to production phase.
+
+# Complex workshop example:
+
+II will not repeat the steps I descirbed earlier for the simple workshop example. I will focus here on the specific aspects related to this  new workshop. Are you familiar with High Performance Computing (HPC)? I am not. Even tough I am surrounded by some experts in that field in the HPE Grenoble Office. Let's consider that one of these colleagues is willing to build up a dedicated workshop on HPC Stax. As usual, we will start by a meeting each of us will explain to the other his goals, and how to achieve them. Once I get a clearer understanding of the technology involved, he and I can move on figure out the best platform to run his woorkshop on.
```

Comment:
Remove the first sentence. It's not required and has a lot of typos. Just start with "I will focus here on..."

The third line down, there's a typo with "though"... "Even though I am..."
Actually, that should be part of the previous short sentence of "I am not, even though I am surrounded by..."

In the second to the last line, modify it to read:
"As usual, we will start with a meeting where each of us will explain to the other what our goals are and how we expect to achieve them."

The last part of the last sentence should read:
"...he and I can move on to figure out what the best platform would be on which to run his workshop."
(watch out for that woorkshop typo)